integration_test/workflow_tests/test_packages/ingest/test_multuple_granules_happy_path.py

```diff
@@ -0,0 +1,197 @@
+import json
+import logging
+import time
+import uuid
+from unittest import TestCase
+
+import boto3
+
+import helpers
+
+
+class TestMultipleGranulesHappyPath(TestCase):
+    def test_multiple_granules_happy_path(self):
+        """
+        If multiple granules are provided, should store them in DB as well as in recovery bucket.
+        """
+        try:
+            # Set up Orca API resources
+            # ---
+            my_session = helpers.create_session()
+            granule_id_1 = uuid.uuid4().__str__()
+            granule_id_2 = uuid.uuid4().__str__()
+            provider_id = uuid.uuid4().__str__()
+            provider_name = uuid.uuid4().__str__()
+
+            collection_name = uuid.uuid4().__str__()
+            collection_version = uuid.uuid4().__str__()
+            bucket_name = "orca-sandbox-s3-provider"  # standard bucket where initial file exists
+            recovery_bucket_name = "rhrh-orca-primary"
+            excluded_filetype = []
+            key_name_1 = "PODAAC/SWOT/ancillary_data_input_forcing_ECCO_V4r4.tar.gz"
```

Comment:
Would prefer these files to be randomized to avoid collisions.

Reply:
the exisitng test assumes the files are already present in the bucket so to randomize the name, we have to first create and copy the files to the bucket before running tests.
Recommend creating additional task for that approach if desired.

Reply:
Recommend discussion with ORCA team. Create card as a placeholder for discussion.
At minimum, this should be documented as it causes tests to fail if cleanup steps are not manually performed.

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-628 to discuss with orca team and implementation.

Updated doc for now.